.env

```diff
@@ -0,0 +1,3 @@
+MONGO_DB_URL=mongodb://127.0.0.1:27017/hg
```

Comment:
Looks like we're using a local DB.

Reply:
That's because the actual remote URL can't be made public